ray-operator/controllers/ray/rayservice_controller_unit_test.go

```diff
@@ -779,6 +782,15 @@ func TestReconcileRayCluster(t *testing.T) {
 			enableZeroDowntime:      false,
 			shouldPrepareNewCluster: true,
 		},
+		// Test 6: If active KubeRay version doesn't match the KubeRay version annotation on the RayCluster, update the RayCluster.
+		"Active RayCluster exists. KubeRay version is mismatched. Update the RayCluster.": {
+			activeCluster:           activeCluster.DeepCopy(),
+			updateRayClusterSpec:    true,
+			enableZeroDowntime:      false,
```

Comment:
How about testing it with `enableZeroDowntime: true`? Zero downtime upgrade is enabled by default, so setting it to true is more close to the real situation.

Reply:
Done in [3142ece](https://github.com/ray-project/kuberay/pull/2320/commits/3142eced7e35e4b5ca18b3f3c7aef586796b7b52)